Files/Views/SettingsPages/About.xaml

```diff
@@ -188,6 +188,8 @@
                             VerticalAlignment="Center"
                             Text="License:" />
                         <HyperlinkButton NavigateUri="https://github.com/QL-Win/QuickLook/blob/master/LICENSE-GPL.txt">GPL</HyperlinkButton>
+                        <Button Content="Check for updates" Click="UpdateCheck_Click">
```

Comment:
Please remove the button and have the code checking for updates run when the app is first launched.

Reply:
@yaichenbaum I don't think it is a good idea to show update dialog on every app launch.

Reply:
There is no need to show a dialog every time the app is opened, the only time a dialog will be displayed is if a mandatory update is detected.

Reply:
i don't think it must be at startup by default. Maybe an option. Noting more, as store is already updating apps

Reply:
@boscorelly This pull request is regarding requiring mandatory updates.